tsl/src/remote/connection.c

```diff
@@ -1024,6 +1026,7 @@ remote_connection_get_result(const TSConnection *conn, TimestampTz endtime)
 				if (PQconsumeInput(conn->pg_conn) == 0)
 				{
 					pgres = PQmakeEmptyPGresult(conn->pg_conn, PGRES_FATAL_ERROR);
+					Ensure(pgres, "out of memory");
```

Comment:
This check makes sense, as opposed to most of the other ones, since this one actually allocates memory.